CHANGELOG.md

```diff
@@ -145,6 +145,9 @@
   [#9737](https://github.com/Kong/kong/pull/9737)
 - Add `key-sets` entity to group and manage `keys`
   [#9737](https://github.com/Kong/kong/pull/9737)
+- Add `debug` option to kong config headers to set `kong-debug`
+  feature, default behavior setting to off
```

Comment:
this should default to `on` to not becoming a breaking change

Reply:
as discussed internally we are okay to move this forward as a breaking change so resolving this